lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/driver/evaluator/EvaluatorResourceManagerErrorHandler.java

```diff
@@ -62,7 +62,11 @@ public void onNext(final RemoteMessage<ReefServiceProtos.RuntimeErrorProto> runt
     if (evaluatorManager.isPresent()) {
       evaluatorManager.get().onEvaluatorException(evaluatorException);
     } else {
-      LOG.log(Level.WARNING, "Unknown evaluator runtime error: " + error);
+      if (this.evaluators.wasClosed(evaluatorId)) {
+        LOG.log(Level.FINE, "Evaluator [" + evaluatorId + "] has raised exception after it was closed.");
```

Comment:
Probably yes, since in the normal scenario exception would happen first, causing evaluator close. Same as in other places in this PR, is anything happens to evaluator after it's closed, it's suspicious.